shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/pom.xml

```diff
@@ -49,8 +49,8 @@
                         <id>antlr</id>
                         <configuration>
                             <libDirectory>src/main/antlr4/imports/mysql/</libDirectory>
-                            <listener>false</listener>
-                            <visitor>false</visitor>
+                            <listener>true</listener>
```

Comment:
Do we need generate listener too?

Reply:
Listeners are used for checking later and not necessary now.